docker-compose.yml

```diff
@@ -1,6 +1,11 @@
 version: '3'
 
 services:
+  test_browser:
+    build:
+      context: .
```

Comment:
probably don't need the `context` because it's the default?

Reply:
fails to run without:
```
ERROR: The Compose file is invalid because:
Service test_browser has neither an image nor a build context specified. At least one must be provided.
```

Reply:
So there's no default, good to know